presto-client/src/main/java/com/facebook/presto/client/OkHttpUtil.java

```diff
@@ -91,6 +91,15 @@ public static Interceptor basicAuth(String user, String password)
                 .build());
     }
 
+    public static Interceptor tokenAuth(String accessToken)
+    {
+        requireNonNull(accessToken, "accessToken is null");
```

Comment:
Should we do validation to make sure the token is only printable ASCII? I'm wondering what happens if you put in a newline, etc.

Reply:
What happens for username and password?

Reply:
The password is base64 encoded. I suppose we should have validation for username and other headers.